src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -68,54 +112,154 @@ export const updateTrackStatesAndSave: ActionCreator<
   browserStorageService.saveTrackStates(trackStates);
 };
 
+export const saveTrackConfigsForGenome: ActionCreator<
+  ThunkAction<void, any, void, Action<string>>
+> = (genomeId: string) => (_, getState: () => RootState) => {
+  const state = getState();
+  const trackConfigsForGenome = state.browser.trackConfig.configs[genomeId];
+  const fieldsForSaving = pick(trackConfigsForGenome, [
+    'shouldApplyToAll',
+    'tracks'
+  ]);
+
+  browserTrackConfigStorageService.setTrackConfigs({
+    genomeId,
+    fragment: fieldsForSaving
+  });
+};
+
+// TODO: get proper data from the backend in order not to hack track id
+export const getTrackType = (trackId: string) => {
+  if (trackId.startsWith('gene')) {
+    return TrackType.GENE;
+  } else {
+    return TrackType.REGULAR;
+  }
+};
+
+const initialState: TrackConfigState = {
+  browserTrackCogs: {
+    cogList: {},
+    selectedCog: null
+  },
+  configs: {}
+};
+
 const browserTrackConfigSlice = createSlice({
   name: 'genome-browser-track-config',
-  initialState: defaultTrackConfigState,
+  initialState: initialState,
```

Comment:
Just `initialState` would do. No need to specify key and value separately when both have the same name.